geoportailv3/static/js/search/searchdirective.js

```diff
@@ -301,6 +365,64 @@ app.SearchDirectiveController =
 
 };
 
+/**
+ * @param {Fuse} fuseEngine
+ * @param {string} searchString
+ * @return {Array.<string>}
+ * @private
+ */
+app.SearchDirectiveController.prototype.matchLayers_ =
+    function(fuseEngine, searchString) {
+  var fuseResults = fuseEngine.search(searchString).slice(0, 5);
+  var results = [];
+  goog.array.forEach(fuseResults, function(element) {
+    results.push(element['item']);
+  });
+  return results;
+};
+
+/**
+ * @param {string} searchString
+ * @return {Array<ol.Feature>}
+ * @private
+ */
+app.SearchDirectiveController.prototype.matchCoordinate_ =
+    function(searchString) {
+  var results = [];
+  var re = {
+    'EPSG:2169': {
+      regex: /(\d{4,5})\s*[E]?\W*(\d{4,5})\s*[N]?/,
+      label: 'LUREF'
+    },
+    'EPSG:4326': {
+      regex:
+          /(\d{1,2}[\,\.]\d{4,5})\d*\s?[E]?\W*(\d{1,2}[\,\.]\d{4,5})\d*\s?[N]?/,
+      label: 'long/lat WGS84'
+    }
+  };
+  var m;
+  for (var epsgCode in re) {
+    if ((m = re[epsgCode].regex.exec(searchString)) !== null) {
+      var easting = parseFloat(m[1]);
+      var northing = parseFloat(m[2]);
+      var point = /** @type {ol.geom.Point} */
+          (new ol.geom.Point([easting, northing])
+         .transform(epsgCode, 'EPSG:3857'));
+      if (ol.extent.containsCoordinate(
+          this.maxExtent_, point.getCoordinates())) {
+        var feature = /** @type {ol.Feature} */
+            (new ol.Feature({
+              geometry: point,
+              'label': easting + 'E ' + northing + 'N ',
+              'epsgLabel': re[epsgCode].label
+            }));
```

Comment:
I'd recommend using `set` to avoid problems:

```
var feature = new ol.Feature(point);
feature.set('label', easting + 'E ' + northing + 'N ');
feature.set('epsgLabel', re[epsgCode].label);
```

See https://github.com/camptocamp/ngeo/blob/master/docs/developer-guide.md#custom-olobject-properties.